tests/cypress/tests/unit/dash_table_queries_test.ts

```diff
@@ -74,8 +74,8 @@ describe('Dash Table Queries', () => {
 
                 describe('contains', () => {
                     processCases(c.syntaxer, [
-                        { name: 'cannot compare "11" to 1', query: `${c.hideOperand ? '' : '{a} '}contains 1`, target: { a: '11' }, valid: true, evaluate: true },
-                        { name: 'cannot compare 11 to 1', query: `${c.hideOperand ? '' : '{a} '}contains 1`, target: { a: 11 }, valid: true, evaluate: false },
+                        { name: 'compares "11" to 1', query: `${c.hideOperand ? '' : '{a} '}contains 1`, target: { a: '11' }, valid: true, evaluate: true },
+                        { name: 'compares 11 to 1', query: `${c.hideOperand ? '' : '{a} '}contains 1`, target: { a: 11 }, valid: true, evaluate: true },
```

Comment:
`contains` now does not distinguish between string 11 and number 11 in all (string|number, string|number) operand/expression combinations